scripts/nf-core

```diff
@@ -94,42 +94,42 @@ def list(keywords, sort, json):
     required = False,
     metavar = "<pipeline name>"
 )
-@click.option(
-    '-i', '--id',
-    help = "ID for web-gui launch parameter set"
-)
 @click.option(
     '-r', '--revision',
     help = "Release/branch/SHA of the project to run (if remote)"
 )
+@click.option(
+    '-i', '--id',
+    help = "ID for web-gui launch parameter set"
+)
 @click.option(
     '-c', '--command-only',
     is_flag = True,
     default = False,
-    help = "Set params directly in the nextflow command."
-)
-@click.option(
-    '-p', '--params-in',
-    type = click.Path(exists=True),
-    help = "Input parameters JSON file."
+    help = "Set params in the nextflow command (no params file)"
```

Comment:
```suggestion
    help = "Set params directly in the Nextflow command i.e. dont create a params file"
```

Reply:
Too long, makes it multiline which looks kind of out-of step with the rest of the options:

```
Options:
  -r, --revision TEXT    Release/branch/SHA of the project to run (if remote)
  -i, --id TEXT          ID for web-gui launch parameter set
  -c, --command-only     Set params directly in the Nextflow command i.e. dont
                         create a params file

  -o, --params-out PATH  Path to save run parameters to
  -p, --params-in PATH   Set of input run params to use from a previous run
  -a, --save-all         Save all parameters, even if unchanged from default
  -h, --show-hidden      Show hidden params which don't normally need changing
  --url TEXT             Customise the builder URL (for development work)
  --help                 Show this message and exit.
```

Reply:
How about this?

```suggestion
    help = "Use Nextflow command-line params (no params file)"
```

Reply:
Ping @drpatelh

Reply:
Sorry. Missed this. Um still a bit unclear to be honest. How about `Create Nextflow command with params (no params file)`

Reply:
```suggestion
    help = "Create Nextflow command with params (no params file)"
```